src/main/java/ch/njol/skript/expressions/ExprBrushableItem.java

```diff
@@ -0,0 +1,104 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.BrushableBlock;
+import org.bukkit.block.BlockState;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Dusted Item")
+@Description({
+	"Represents the item that is uncovered when dusting.",
+	"The only blocks that can currently be \"dusted\" are Suspicious Gravel and Suspicious Sand."
+})
+@Examples({
+	"send target block's brushable item",
+	"set {_gravel}'s brushable item to emerald"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.20+")
+public class ExprBrushableItem extends SimpleExpression<ItemStack> {
+
+	private static final boolean SUPPORTS_DUSTING = Skript.classExists("org.bukkit.block.BrushableBlock");
+
+	static {
+		if (SUPPORTS_DUSTING)
+			Skript.registerExpression(ExprBrushableItem.class, ItemStack.class, ExpressionType.SIMPLE,
+				"[the] %blocks% brush[able] item",
+				"%blocks%'[s] brush[able] item");
```

Comment:
You probably want this to be a property pattern, the first part's sort of wrong right now.
```suggestion
				"[the] brush[able] item of %blocks%",
				"%blocks%'[s] brush[able] item");
```

I'm not sure `brush` on its own is a great pattern for this. If you wanted an alternative maybe `buried item`? I'm not sure.
```suggestion
				"[the] (brushable|buried) item of %blocks%",
				"%blocks%'[s] (brushable|buried) item");
```

You may also want to allow items plural for blocks plural, although I wonder if that could cause confusion since each block only has the one item. Open to feedback.

Reply:
i agree that the current syntax should be changed